python/labbox_ephys/api/_workersession.py

```diff
@@ -71,28 +73,21 @@ def handle_message(self, msg):
             self._readonly = msg['clientInfo']['readOnly']
             if not self._feed_uri:
                 self._feed_uri = 'feed://' + self._default_feed_id
-                # self._feed_uri = kp.create_feed(feed_name='labbox-ephys-default').get_uri()
-            # assert self._feed_uri.startswith('sha1://'), 'For now, feedUri must start with sha1://'
             self._feed = kp.load_feed(self._feed_uri)
-            for key in ['recordings', 'sortings']:
```

Comment:
Now recordings and sortings subfeeds are not treated as special watches... the client now has control for adding arbitrary subfeed watches (rather than the server). This puts us in a position of implementing workspace view in jupyter.